lib/db/DB.ts

```diff
@@ -58,8 +58,22 @@ class DB {
       const { dialectName } = this.sequelize.connectionManager;
       this.logger.info(`connected to database. host:${host} port:${port} database:${database} dialect:${dialectName}`);
     } catch (err) {
-      this.logger.error('unable to connect to the database', err);
-      throw err;
+      if (err instanceof Sequelize.InvalidConnectionError) {
```

Comment:
Trying to run it, this returns `false` when the database doesn't exists. The error type is `SequelizeConnectionError`.

Reply:
Same for me.